source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -77,6 +77,8 @@ If your PSP is Stripe, you can sign in to the [GOV.UK Pay admin tool](https://se
 - payments ('payouts') that Stripe has made to your bank account
 - which of your users' payments are in each payout
 
+From 1 April 2022, [if Stripe charges fees on a failed payment](/reporting/#net-amount) they will deduct the fees from your Stripe balance. The failed payments are bundled into your regular payouts. Until 1 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments.
```

Comment:
```suggestion
From 1 April 2022, [if Stripe charges fees on a failed payment](/reporting/#net-amount), they will deduct the fees from your Stripe balance. The failed payments are bundled into your regular payouts. Until 1 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments.
```
Also, should we change "charges" to "charge", if we're referring to Stripe plurally ("they")?

Reply:
Re

> The failed payments are bundled...

Guessing there's no way we can make this active without opening can of worms/writing something that would sound unnatural?

Reply:
I think there's a way to make it active.

> Stripe bundles the failed payments into...

I'll add it in now. Thanks, Eoin!